### DISCUSSION TITLE
Multiselect - Button in Option

### DISCUSSION THREAD

**Discussion by leonlazer** [vueform/vueform#202] [Questions] [answered]
Hey,
I'm using the Mutliselect to select and add categories for products in my shopbackend. I would also like to have the option to delete them directly through the Multiselect, so I don't have to create a new Page to do just that. Theoretically it works, but it is very fiddely. My Mulitselect looks like this:
```vue
<Multiselect
  :classes="{
    tag: 'bg-orange-50 border border-orange-500 rounded-lg text-sm text-orange-500 py-1 pl-2 mr-1 mb-1 flex items-center whitespace-nowrap min-w-0 rtl:pl-0 rtl:pr-2 rtl:mr-0 rtl:ml-1',
    tagRemove: 'multiselect-tag-remove hover:bg-transparent' ,
    optionSelected : 'bg-primary',
    optionSelectedPointed: 'bg-red-500',
    containerActive: 'ring-0 ring-opacity-0',
  }"
  class="mb-4 min-h-14"
  v-model="formData.categories"
  mode="tags"
  v-bind="categorySelectOptions"
  tag-placeholder="Neue Kategorie einfügen"
  placeholder="Kategorie suchen oder eine neue hinzufügen"
  :multiple="true"
  :taggable="true"
  track-by="code"
  :value="null"
  :closeOnSelect="false"
  :searchable="true"
  :createOption="true"
>
  <template v-slot:option="{ option }">
    <div v-if="option.__CREATE__">
      + {{ option.label }} erstellen
    </div>
    <div v-else class="relative w-full flex justify-between">
    {{ option.label }}
      <DropdownMenu>
        <form @submit.prevent="deleteCategory(option.value)">
          <button :title="option.label" class="py-2 px-5 w-full text-base text-red-600 text-left transition hover:bg-slate-50 flex gap-2 items-center" type="submit"><IconBin/> Kategorie löschen</button>
        </form>
      </DropdownMenu>
    </div>
  </template>
</Multiselect>
```
Which results in this:
![multiselect](https://github.com/vueform/vueform/assets/127108093/b21561f4-d7f5-432d-ab02-43895972071d)

Basically that's fine for me, but the problem is, the moment I click on the "Kategorie löschen" button, it selects the category in 80% of the cases and only sometimes run the function it should run. Is there any way to integrate a Button into an option in a better way, so that the moment you hover the button, all Options are "disabled" until you either click it or move your mouse away from the button?

I hope you understand what my problem is :D Thanks in advance!

**Comment by adamberecz** [comment, marked as answer]
Hi, try adding `@mousedown.stop` to `<button>`:

```vue
<button
  :title="option.label"
  class="py-2 px-5 w-full text-base text-red-600 text-left transition hover:bg-slate-50 flex gap-2 items-center"
  type="submit"
  @mousedown.stop
><IconBin/> Kategorie löschen</button>
        
```

**Reply by leonlazer** [reply]
That was such an easy solution, works perfectly 
Thank you
